index.html

```diff
@@ -71,7 +76,7 @@ <h2>Tonic</h2>
         </section>
     </div>
     
-    <div class="tonic-card">
+    <div class="tonic-card-2">

```

Comment:
- [ ] [OPTIONAL] Please change your work section background color into grey as defined on Figma template. Check the screenshot below :point_down: 

![179305098-72f5e43c-2686-4aa6-a183-cb6902b44044](https://user-images.githubusercontent.com/6615394/179324083-4467f437-aa6e-48eb-9a02-40ec98017af1.png)